usr/sbin/registercloudguest

```diff
@@ -170,6 +173,21 @@ def cleanup():
     utils.clear_new_registration_flag()
     utils.clean_framework_identifier()
 
+# ----------------------------------------------------------------------------
+def setup_registry(registration_target, user, password):
+    """
+    Run the registration process for the container registry
+    """
+    registry_fqdn = registration_target.get_registry_FQDN()
+    if not utils.is_registry_registered(registry_fqdn)
+       and not utils.setup_registry(registry_fqdn, user, password):
+        cleanup()
+        print(
+            'Registration failed(registry), see {0} for details'.format(
+            ), file=sys.stderr
```

Comment:
Syntax error, the substitution is missing, also fixed in #187

Reply:
flake8 is not even called for this code... my bad this should not have landed in any PR. Thanks

Reply:
Thanks I merged your PR and added the code path to flake8 which complained at several places and I fixed those in another commit